packs/docker/actions/lib/docker_wrapper.py

```diff
@@ -1,10 +1,7 @@
 import sys
+import traceback
 
-try:
-    import simplejson as json
-except:
-    import json
-
+from requests.packages.urllib3.exceptions import ReadTimeoutError
```

Comment:
I am not explicitly adding this to requirements but docker-py does depend on this. I can go either ways.